highfidelity/hifi
# Show Server Console Version

This PR displays the server console version.

To test: 
1. Install this build and start the server console.
2. Right click and observe the version number in the list!
